src/components/ui/Collapsible/Collapsible.tsx

```
@@ -11,49 +20,78 @@ import ButtonPrimitive from '~/core/primitives/Button';
+          {/* Default value to be shown */}
+          {defaultOpen && (
+            <CollapsibleItem>{defaultOpen.content}</CollapsibleItem>
+          )}
+          {/* Collapsable Content  */}
+          <CollapsibleContent state={props.open ?? open}>
+            {items.map((item) => (
+              <>
+                {item != defaultOpen && (
+                  <CollapsibleItem>{item.content}</CollapsibleItem>
+                )}
+              </>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)